### DISCUSSION TITLE
server lag

### DISCUSSION THREAD

**Discussion by kevinohashi** [mbround18/valheim-docker#147] [Q&A] [answered]
Wondering if anyone is running into laggy server performance issues with de-sync/rubberbanding? Any solutions/ideas to fix this would be appreciated. It's running on beefy dedicated hardware with a good network connection, but it's happening to players on three continents at once, so it seems server related.

**Comment by mbround18** [comment, marked as answer]
Yes, this is expected because Valheim utilized peer to peer multiplayer technology. My friends and I all have the same issue and the servers a beef cake hosted in a nearby data center. Type `ping` into the F5 console and take a gander. That is your ping to the server but you also have a ping compared to everyone around you.

**Reply by AtroposOrbis** [reply]
#149 This could provide some relief for us! :D @kevinohashi
